src/packages/@ncigdc/components/Project.js

```diff
@@ -110,7 +110,7 @@ const Project = (
           <DownloadButton
             disabled={!biospecimenCount}
             filename={`biospecimen.project-${projectId}`}
-            url={urlJoin(process.env.REACT_APP_API, 'cases')}
+            url={urlJoin(process.env.REACT_APP_GDC_AUTH, 'api/cases')}
```

Comment:
would `REACT_APP_GDC_AUTH_API` ever diverge from `REACT_APP_GDC_AUTH`? maybe it would be worth adding a constants file and just adding `/api` there?